types/src/block/output/alias.rs

```diff
@@ -204,7 +204,7 @@ impl AliasOutputBuilder {
     }
 
     ///
-    pub fn finish(self, token_supply: u64) -> Result<AliasOutput, Error> {
+    fn _finish(self) -> Result<AliasOutput, Error> {
```

Comment:
Just use `finish_unverified` and remove the current `finish_unverified`

Reply:
Yes sir!